OsmAnd/src/net/osmand/plus/views/OsmandMapLayer.java

```diff
@@ -30,6 +30,23 @@ public void onRetainNonConfigurationInstance(Map<String, Object> map) {}
 	
 	public void populateObjectContextMenu(Object o, ContextMenuAdapter adapter) {}
 
+	/**
+	 * return true when this layer should handle single tap instead of context layer
+	 * @return
+	 */
+	public boolean isSingleTapResponder() {
+		return false;
+	}
+
+	/**
+	 * return true when context layer should open context menu of the long pressed point
+	 * (instead of common behaviour when lat/lon text opens on long press)
+	 * @return
+	 */
+	public boolean shouldProcessLongPress() {
```

Comment:
Similar suggestion to isSingleTapResponder(), so we don't overload map layer with methods.